### ISSUE TITLE
About discriminator

### ISSUE THREAD

**Issue by takdcloose** [clovaai/stargan-v2#71]
It seems not to use any activation functions like sigmoid in the last layer of the discriminator. Is it intended? In my knowledge, It happens when using binary cross-entropy

**Comment by takdcloose** [comment]
it's included in binary_cross_entropy_with_logits. forget about this issue

**Comment by yunjey** [comment]
@takdcloose As you mentioned, `binary_cross_entropy_with_logits `do not require a sigmoid input. This loss combines the sigmoid function and the binary cross entropy loss in a single operation using log-sum-exp trick. This is more numerically stable than using a plain sigmoid followed by a binary cross entropy.
